rpcs3/main_application.cpp

```diff
@@ -50,10 +64,60 @@ void main_application::InitializeEmulator(const std::string& user, bool show_gui
 	sys_log.always()("%s", firmware_string);
 }
 
+void main_application::OnEmuSettingsChange()
+{
+	if (Emu.IsRunning())
+	{
+		if (g_cfg.misc.prevent_display_sleep)
```

Comment:
This whole section is weird. Maybe the function names should be switched if the functionality is correct.
I'm referring to the fact that enable_display_sleep is called when prevent_display_sleep is enabled 🤦

Reply:
this is just moved code

Reply:
it's probably a bug? idk. I never used it and no one ever complained

Reply:
I switched them. Looked like a bug to me as well.